backends/openvino/runtime/OpenvinoBackend.cpp

```diff
@@ -18,35 +17,13 @@
 #include <executorch/runtime/core/exec_aten/util/dim_order_util.h>
 #include <executorch/runtime/core/exec_aten/util/scalar_type_util.h>
 
-#include "OpenvinoBackend.hpp"
-
-using namespace std;
-using executorch::aten::ScalarType;
-using executorch::runtime::ArrayRef;
-using executorch::runtime::Backend;
-using executorch::runtime::BackendExecutionContext;
-using executorch::runtime::BackendInitContext;
-using executorch::runtime::CompileSpec;
-using executorch::runtime::DelegateHandle;
-using executorch::runtime::Error;
-using executorch::runtime::EValue;
-using executorch::runtime::FreeableBuffer;
-using executorch::runtime::MemoryAllocator;
-using executorch::runtime::Result;
+#include "OpenvinoBackend.h"
 
 namespace executorch {
 namespace backends {
 namespace openvino {
 
-OpenvinoBackend::OpenvinoBackend() {
-  if (!is_available()) {
-    // ET_LOG(Error, "OpenVINO runtime is not available. Initialization
-    // failed.");
-    throw std::runtime_error("OpenVINO runtime not available");
-  }
-
-  // ET_LOG(Info, "OpenVINO runtime successfully verified and initialized.");
-}
```

Comment:
With the latest static lib changes, do you still need this function to be removed?

Reply:
I'm testing this with latest updates.

Reply:
I still observe the same issue. "is_available" call stalls in openvino side (in get_devices call) if it is called in constructer.

Reply:
ok, we can leave it for now. Thanks for checking